cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -711,8 +713,10 @@ public MapAndStatus(T loadMap, K status) {
                                                     KafkaRebalance kafkaRebalance,
                                                     KafkaRebalanceAnnotation rebalanceAnnotation,
                                                     RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder) {
-        if (rebalanceAnnotation == KafkaRebalanceAnnotation.refresh) {
-            // The user has fixed the error on the resource and want to 'refresh'
+
+        if (rebalanceAnnotation == KafkaRebalanceAnnotation.refresh || disabled) {
```

Comment:
Wouldn't we want this conditional to be:
```
rebalanceAnnotation == KafkaRebalanceAnnotation.refresh && !disabled
``` 
So that a new request isn't made when CC is disabled?